sys/linux/dev_floppy.txt

```diff
@@ -9,7 +9,14 @@ include <uapi/linux/fd.h>
 
 resource fd_floppy[fd]
 
-syz_open_dev$floppy(dev ptr[in, string["/dev/fd#"]], id intptr, flags flags[open_flags]) fd_floppy
+# O_ACCMODE is used for ioctl-only opens, see FMODE_WRITE_IOCTL
+# https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/include/linux/fs.h?h=v5.14#n129
+# Examples:
+# - https://sources.debian.org/src/fdutils/5.6-2/src/floppycontrol.c/?hl=289#L289
+# - https://sources.debian.org/src/fdutils/5.6-2/src/diskd.c/?hl=90#L90
+fd_open_flags = open_flags, O_ACCMODE
```

Comment:
I don't think extending flags in this way is supported.
This should be treated as kernel has some const named open_flags. Non-existent consts are silently ignored (some arches/kernels don't have all consts). But if a const is not present on all arches, it should produce an error during `make generate` IIRC. Don't you see such an error?.. or do I mis-interpret what happens in this case?

Reply:
Added rest of the flags to fd_open_flags. Dropped only O_CREAT, O_DIRECTORY, O_EXCL, O_LARGEFILE, O_NOCTTY, O_NOFOLLOW, O_PATH and __O_TMPFILE.